client/state/selectors/get-onboarding-url.js

```diff
@@ -32,10 +31,7 @@ export default function getOnboardingUrl( state ) {
 		return config( 'gutenboarding_url' );
 	}
 
-	const existingUsersOnboardingVariant = getVariationForUser(
-		state,
-		'new_onboarding_existing_users_non_en_v5'
-	);
+	const existingUsersOnboardingVariant = null;
```

Comment:
Lines 34-37 don't do anything useful now and line 36 is unreachable. Remove?

Reply:
I want to leave it up to experimenters to remove the actual experiment code.